e2e/atlas/clusters_test.go

```diff
@@ -73,14 +67,8 @@ func TestClustersFlags(t *testing.T) {
 		)
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
-
-		if err != nil {
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
-		}
-
-		if !strings.Contains(string(resp), "Cluster available") {
-			t.Errorf("got=%#v\nwant=%#v\n", string(resp), "Cluster available at:")
-		}
+		assert.NoError(t, err)
```

Comment:
Are we shifting from using `if err!= nil { t.Fatalf(...)}` to `assert.NoError`? If so, could you please provide information about this decision on the PR description? If not, could you explain the changes to this file?

Reply:
This shifting has already begun in the e2e tests (examples https://github.com/mongodb/mongocli/blob/master/e2e/atlas/alerts_test.go,
https://github.com/mongodb/mongocli/blob/master/e2e/atlas/custom_dns_test.go
https://github.com/mongodb/mongocli/blob/master/e2e/atlas/search_test.go). We generally try to use `assert` in the e2e tests where we have several `if` to make the code more readable. I suppose we could use it for unit tests as well but the impact would be less noticeable.